### PR TITLE
New package: thinkpad-scripts-4.11.0.

### PR DESCRIPTION
**Pull request by non-Jedi**
Since this is only useful on the specific Thinkpad models in question, should I set `only_for_archs="x86_64 x86_64-musl i686 i686-musl"`?

### CODE REVIEW

File: srcpkgs/thinkpad-scripts/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+maintainer="Adam Beckmeyer <adam_gpg@thebeckmeyers.xyz>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/martin-ueding/thinkpad-scripts"
+distfiles="https://github.com/martin-ueding/thinkpad-scripts/archive/v${version}.tar.gz"
+checksum=a24bf13793d02348778465e8e608429570a2ce676e1d15f57da69beff2697684
+
+post_install() {
+	python3 setup.py install --single-version-externally-managed --root="${DESTDIR}"
+	mv "${DESTDIR}/lib/udev" "${DESTDIR}/usr/lib/"
+	rmdir "${DESTDIR}/lib"
```
Comment (maxice8) [inline srcpkgs/thinkpad-scripts/template]: no need, this directory will be removed automatically.

File: srcpkgs/thinkpad-scripts/template [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'thinkpad-scripts'
+pkgname=thinkpad-scripts
+version=4.11.0
+revision=1
+build_style=gnu-makefile
```
Comment (maxice8) [inline srcpkgs/thinkpad-scripts/template]: shouldn't this be a python3-module then ?

Comment (non-Jedi) [reply srcpkgs/thinkpad-scripts/template]: Honestly, I didn't look closely enough to have any idea how the internals of this package work. I saw that it had a Makefile and just started with that and then later saw the installation instructions wanted you to do something with setup.py. I'll see if I can package it as `build_style=python3-module` tomorrow.

**Comment by non-Jedi** [comment]
Pushed a commit addressing your comments @maxice8. Looks like I got your handle wrong in the commit message; sorry.

Let me know if it looks good, and you'd like me to squash and force push.

**Comment by maxice8** [comment]
Please squash them, all changes to a package should be in one commit

**Pull request closed without merging**